### PR TITLE
Add Support for VS code editor 

### PR DESCRIPTION
**Pull request by Abdullamhd**
Addd new Resources in this file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,4 @@
 ### Fixed a new issue 
 
 ### This documentation is fixed 
+## A new file system has been added to the git
```
Comment (MLVisionPlatform) [inline README.md:8]: This line should be removed in future release

Comment (Abdullamhd) [reply README.md:8]: this file will be removed in next release

Comment (MLVisionPlatform) [reply README.md:8]: this code has been changed based on this pull request 
#8

**Review by MLVisionPlatform — APPROVED** [review]
this changes is done

**Pull request merged**